.gitattributes

```diff
@@ -0,0 +1,25 @@
+## Platform files generated by Flutter during `flutter create`
+**/example/android/ linguist-generated=true
+**/example/ios/ linguist-generated=true
+**/example/linux/ linguist-generated=true
+**/example/macos/ linguist-generated=true
+**/example/web/ linguist-generated=true
+**/example/windows/ linguist-generated=true
+
+## Generated by mono_repo
+.github/workflows/build.yml linguist-generated=true
+.github/workflows/lint.yml linguist-generated=true
+.github/workflows/test.yml linguist-generated=true
+.github/workflows/native_test.yml linguist-generated=true
+tool/ci.sh linguist-generated=true
+
+## Generated by Pigeon
+**/pigeons/*.java linguist-generated=true
+**/pigeons/*.h linguist-generated=true
+**/pigeons/*.m linguist-generated=true
+
+## Generated SDK files
+packages/**/src/lib/sdk/src linguist-generated=true
```

Comment:
```suggestion
packages/**/lib/src/sdk/ linguist-generated=true
```

Reply:
Oops. Mixed up `lib` and `src` here. Will update.

Re: trailing `src` - Some files under `sdk` are not generated like `sdk_exception.dart`. Maybe we move those to a new folder (`/src/bridge`?) which I think would allow us to exclude them.